Libraries/PeriphDrivers/Source/GPIO/gpio_ai85.c

```diff
@@ -109,6 +109,11 @@ int MXC_GPIO_Config(const mxc_gpio_cfg_t *cfg)
 
     MXC_GPIO_Init(1 << port);
 
+    if (MXC_GPIO_GetConfigLock() == MXC_GPIO_CONFIG_LOCKED) {
+        // Configuration is locked.  Ignore any attempts to change it.
+        return E_NO_ERROR;
```

Comment:
@lorne-maxim what do you think about returning `E_BAD_STATE` instead of no error?

Reply:
I had considered it, but the problem is our PeriphInit functions check the return value of this function.  So someone that has used the PinMux tool will have their configuration locked.  They will then have no way of calling the PeriphInit function that won't return an error.